packages/contracts-bedrock/src/L1/SystemConfig.sol

```diff
@@ -122,16 +123,22 @@ contract SystemConfig is OwnableUpgradeable, ISemver {
     /// @notice The EIP-1559 elasticity multiplier.
     uint32 public eip1559Elasticity;
 
+    /// @notice The operator fee scalar.
+    uint32 public operatorFeeScalar;
+
+    /// @notice The operator fee constant.
+    uint64 public operatorFeeConstant;
+
     /// @notice Emitted when configuration is updated.
     /// @param version    SystemConfig version.
     /// @param updateType Type of update.
     /// @param data       Encoded update data.
     event ConfigUpdate(uint256 indexed version, UpdateType indexed updateType, bytes data);
 
     /// @notice Semantic version.
-    /// @custom:semver 2.4.0
+    /// @custom:semver 2.4.0-beta.1
     function version() public pure virtual returns (string memory) {
-        return "2.4.0";
+        return "2.4.0-beta.1";
```

Comment:
Are we still appending -beta? I know there was some discussion of just using plain versions and it's what OPCM deploys that defines what's officially released or not.

Reply:
https://devdocs.optimism.io/contracts-bedrock/policies/versioning.html#individual-contract-versioning -> no beta tag